docs/development/reviewing-patches.rst

```diff
@@ -34,6 +34,47 @@ These are small things that are not caught by the automated style checkers.
 * Does a variable need a better name?
 * Should this be a keyword argument?
 
+Testing branches remotely
+-------------------------
+
+It can be helpful to check out a branch with a PR on it, in order to test
+how the changes in it work on your machine. To do so, first pull
+down the PR branch by running the following commands:
+
+.. code-block:: console
+
+  $ git checkout master
+  $ git fetch upstream pull/ID/head:BRANCHNAME
+
+In this example, upstream refers to the Warehouse repo on github.
```

Comment:
I suppose there's a chance the person reading this doesn't have a remote set yet for the main Warehouse repo. I suggest you add an instruction saying that if they don't yet have a remote set up for the main Warehouse repository, they can set up one called "upstream" by running:

``git remote add upstream git@github.com:pypa/warehouse.git``

Also, 

> upstream refers to the Warehouse repo on github.

Please format the word "upstream"  as ``upstream`` to make it clearer that it's a chunk of code, and change "github" to "GitHub" (watch the capitalization).